pytradfri/__main__.py

```diff
@@ -14,9 +18,21 @@
 CONFIG_FILE = "tradfri_standalone_psk.conf"
 
 
-def main(org_args):
-    """Run main."""
-    global devices, homekit_id, light, api, lights, gateway, groups, moods, tasks, dump_devices, dump_all  # pylint: disable=global-variable-undefined, invalid-name
+if __name__ == "__main__":
```

Comment:
By putting all the code here instead of in a function, we don't need the `global` definition, which was messing up the type hints.